From rdunlap@xenotime.net Sat Aug 26 10:55:29 2006
Date: Sat, 26 Aug 2006 10:56:10 -0700
From: "Randy.Dunlap" <rdunlap@xenotime.net>
Cc: gregkh <greg@kroah.com>, naranjo.manuel@gmail.com, akpm <akpm@osdl.org>
Subject: aircable: fix printk format warnings
Message-Id: <20060826105610.9bfd4060.rdunlap@xenotime.net>

From: Randy Dunlap <rdunlap@xenotime.net>

Fix printk format warnings:
drivers/usb/serial/aircable.c:221: warning: format ‘%Zd’ expects type ‘signed size_t’, but argument 4 has type ‘int’
drivers/usb/serial/aircable.c:283: warning: format ‘%Zd’ expects type ‘signed size_t’, but argument 4 has type ‘int’

Signed-off-by: Randy Dunlap <rdunlap@xenotime.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/aircable.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- gregkh-2.6.orig/drivers/usb/serial/aircable.c
+++ gregkh-2.6/drivers/usb/serial/aircable.c
@@ -218,7 +218,7 @@ static void aircable_send(struct usb_ser
 
 	buf = kzalloc(count + HCI_HEADER_LENGTH, GFP_ATOMIC);
 	if (!buf) {
-		err("%s- kzalloc(%Zd) failed.", __FUNCTION__,
+		err("%s- kzalloc(%d) failed.", __FUNCTION__,
 		    count + HCI_HEADER_LENGTH);
 		return;
 	}
@@ -280,7 +280,7 @@ static void aircable_read(void *params)
 
 	tty_prepare_flip_string(tty, &data, count);
 	if (!data){
-		err("%s- kzalloc(%Zd) failed.", __FUNCTION__, count);
+		err("%s- kzalloc(%d) failed.", __FUNCTION__, count);
 		return;
 	}
 
